main_f4.c

```diff
@@ -262,6 +262,33 @@ board_test_force_pin()
 	return false;
 }
 
+#if INTERFACE_USB
+#if !defined(BOARD_USB_VBUS_SENSE_DISABLED)
+static bool
+board_test_usb_vbus()
+{
+  bool ret = false;
+
+  /* initialise Port A GPIO9 to sample VBUS */
+  rcc_peripheral_enable_clock(&RCC_AHB1ENR, RCC_AHB1ENR_IOPAEN);
```

Comment:
Can PORT A be uses ad other GPIO in the boot loader?

Reply:
As far as I know, PA9 is only used for the VBUS sensing. I didn't disable, the RCC clock, since when the board_deinit() is called the whole RCC_AHB1ENR is reset to its default value.

Reply:
I guess since it was wrapped in the "if INTERFACE_USB" this change is safe.